lib/services/devices/deviceRegistryMongoDB.js

```diff
@@ -287,6 +288,7 @@ function update(device, callback) {
             data.endpoint = device.endpoint;
             data.name = device.name;
             data.type = device.type;
+            data.apikey = device.apikey;
```

Comment:
Does documentation any update regarding this? Not sure if we have documented in some place the payload of the update operations, in which case I understand apiKey should be now included.

Reply:
apikey currently is in device model: https://github.com/telefonicaid/iotagent-node-lib/blob/master/doc/api.md#device-model
but no full options are describe in update (PUT) request: https://github.com/telefonicaid/iotagent-node-lib/blob/master/doc/api.md#put-iotdevicesdeviceid

Reply:
Good enough. NTC